_diagrams/README.md

```diff
@@ -0,0 +1,113 @@
+The `_diagrams` folder is only used to store diagram source files and will be ignored by Jekyll 
+when building the site.
+
+# Tools
+
+The diagram was created in [Sketch](https://www.sketch.com/) and exported using the 
+[Sketch Interactive Export](https://github.com/mathisonian/sketch-interactive-export) plugin, 
+which allows creating class names inside the exported SVG.
+
+# How it works
+
+## Add classes in Sketch
+
+Add a layer or group name like `.g-caption`. 
+The resulting SVG markup will contain `class="g-caption"`.
+
+To have several classes for the one element add `.g-caption.command-dispatcher`. 
+The result will be `class="g-caption command-dispatcher"`.
+
+A layer name without a prefixed period will be used as ID.
+For example, `g-caption.command-dispatcher` will result `id="g-caption" class="command-dispatcher"`.
+
+## Export SVG
+
+1. Select the diagram artboard.
+2. Go to the right sidebar;
```

Comment:
Let's make it consistent and use `.` at the end of the points. It's here and also below.